std_msgs/Header header

uint32 ID
## 控制命令的模式 
uint8 CMD
# enum CMD 控制模式枚举
uint8 HOLD=0
uint8 RIGHT=1
uint8 BACK=2
uint8 LEFT=3
uint8 FORWARD=4
uint8 RESET=99
